scripts/exrrfs_bufrsnd.sh

```diff
@@ -252,13 +258,15 @@ $OUTFILDYN
 $OUTFILPHYS
 EOF
 
-#  export FORT19="$DATA/bufrpost/regional_profdat"
-#  export FORT79="$DATA/bufrpost/profilm.c1.${tmmark}"
-#  export FORT11="./itag"
-
-ln -sf $DATA/bufrpost/regional_profdat     fort.19
-ln -sf $DATA/bufrpost/profilm.c1.${tmmark} fort.79
-ln -sf ./itag                              fort.11
+  if [[ "${NET}" = "RTMA"* ]]; then
+    ln -sf $DATA/bufrpost/regional_profdat     fort.19
+    ln -sf $DATA/bufrpost/profilm.c1.${tmmark} fort.79
+    ln -sf ./itag                              fort.11
+  else
+    export FORT19="$DATA/bufrpost/regional_profdat"
+    export FORT79="$DATA/bufrpost/profilm.c1.${tmmark}"
+    export FORT11="./itag"
+  fi
```

Comment:
Just curious why the rrfs-version bufrsnd needs those FORT variables but rtma-version does not need?